docs/_docs/hivemq/policies.md

```diff
@@ -0,0 +1,290 @@
+---
+nav_order: 1
+redirect_from: /docs/hivemq/policies.html
+---
+
+# Policies
+
+***
+
+The policies command of the HiveMQ command line offers a set of commands to work with policies using a HiveMQ Data
+Validation API endpoint.
+
+```
+mqtt hivemq policies
+```
+
+***
+
+## Commands
+
+| Command | Explanation                         |
+|---------|-------------------------------------|
+| get     | See [Get Policy](#get-policy)       |
+| delete  | See [Delete Policy](#delete-policy) |
+| create  | See [Create Policy](#create-policy) |
+| list    | See [List Policies](#list-policies) |
+
+***
+
+## Options
+
+### Help Options
+
+{% include options/help-options.md defaultHelp=true %}
+
+***
+
+# Get Policy
+
+*** 
+
+Retrieve a single existing policy.
+
+```
+mqtt hivemq policies get
+```
+
+***
+
+## Simple Example
+
+```
+$ mqtt hivemq policies get --id my-policy-id 
+
+{
+  "createdAt": "2023-05-25T14:58:09.093Z",
+  "id": "my-policy-id",
+  "matching": {
+    "topicFilter": "topic/1"
+  },
+  "onFailure": {
+    "pipeline": []
+  },
+  "onSuccess": {
+    "pipeline": []
+  },
+  "validation": {
+    "validators": [
+      {
+        "arguments": {
+          "strategy": "ALL_OF",
+          "schemas": [
+            {
+              "schemaId": "my-schema"
+            }
+          ]
+        },
+        "type": "schema"
+      }
+    ]
+  }
+}
+```
+
+***
+
+## Options
+
+| Option | Long Version | Explanation                    | Default  |
+|--------|--------------|--------------------------------|----------|
+| `-i`   | `--id`       | The id of the policy to fetch. | required |
+
+### API Connection Options
+
+{% include options/api-connection-options.md %}
+
+### Logging Options
+
+{% include options/logging-options.md %}
+
+### Help Options
+
+{% include options/help-options.md defaultHelp=true %}
+
+***
+
+# Delete Policy
+
+*** 
+
+Delete a single existing policy.
+
+```
+mqtt hivemq policies delete
+```
+
+***
+
+## Simple Example
+
+```
+$ mqtt hivemq policies delete --id my-policy-id 
```

Comment:
There is a style-guide that suggests commands without sout to be written without a $ and those with return text to be written with a $.